scripts/extrinsic_check.sh

```diff
@@ -24,7 +24,7 @@ latest_release() {
 latest_release=$(latest_release 'gluwa/creditcoin')
 RELEASE_BIN="./creditcoin-node"
 echo "[+] Fetching binary for Creditcoin version $latest_release"
-curl -L "https://github.com/gluwa/creditcoin/releases/download/$latest_release/creditcoin-v${latest_release}-x86_64-unknown-linux-gnu.tar.gz"  --output creditcoin.tar.gz && tar -xzf creditcoin.tar.gz || exit 1
+curl -L "https://github.com/gluwa/creditcoin/releases/download/$latest_release/creditcoin-v${latest_release}-x86_64-unknown-linux-gnu.tar"  --output creditcoin.tar && tar -xzf creditcoin.tar || exit 1
```

Comment:
For most platforms the download artifacts are compressed. IMO we should change the pipeline which builds the Linux artifacts to compress them and push a tar.gz file instead of changing the script which consumes them.